### PR TITLE
Spreadsheet column name change for principal_work

### PR DESCRIPTION
**Pull request by longhotsummer**

### CODE REVIEW

File: indigo_app/views/works.py [hunk 1]
```diff
@@ -652,8 +652,8 @@ def get_works(self, table, form):
                             info['repealed_by'] = row.get('repealed_by')
                         if row.get('with_effect_from'):
                             info['with_effect_from'] = row.get('with_effect_from')
-                        if row.get('parent_work'):
-                            info['parent_work'] = row.get('parent_work')
+                        if row.get('principal_work'):
```
Comment (goose-life) [inline indigo_app/views/works.py]:
```suggestion
                        if row.get('primary_work'):
```

File: indigo_app/views/works.py [hunk 2]
```diff
@@ -652,8 +652,8 @@ def get_works(self, table, form):
...
+                            info['parent_work'] = row.get('principal_work')
```
Comment (goose-life) [inline indigo_app/views/works.py]:
```suggestion
                            info['parent_work'] = row.get('primary_work')
```

**Comment by longhotsummer** [comment]
OFFS! 🤦‍♂

**Comment by goose-life** [comment]
principal (main work) vs amendment
primary (parent work) vs subsidiary

**Pull request merged**